Allow build without libxml2.  Do we want that?

cvimportdlg.o: In function `ImportSVG':
cvimportdlg.c:(.text+0x14a): undefined reference to `SCImportSVG'
cvimportdlg.o: In function `ImportGlif':
cvimportdlg.c:(.text+0x189): undefined reference to `SCImportGlif'
collect2: ld returned 1 exit status
make[3]: *** [fontforge] Fout 1
make[3]: Map '/home/janneke/vc/gub/target/tools/build/fontforge-20080927/fontforge' wordt verlaten
make[2]: *** [fontforge] Fout 2
make[2]: Map '/home/janneke/vc/gub/target/tools/build/fontforge-20080927' wordt verlaten

--- fontforge-20080927/fontforge/cvimportdlg.c~	2008-08-19 01:22:41.000000000 +0200
+++ fontforge-20080927/fontforge/cvimportdlg.c	2008-11-10 20:41:06.000000000 +0100
@@ -60,9 +60,11 @@
     fclose(plate);
 }
 
+#ifndef _NO_LIBXML
 static void ImportSVG(CharView *cv,char *path) {
     SCImportSVG(cv->b.sc,CVLayer((CharViewBase *) cv),path,NULL,0,false);
 }
+#endif
 
 static void ImportGlif(CharView *cv,char *path) {
     SCImportGlif(cv->b.sc,CVLayer((CharViewBase *) cv),path,NULL,0,false);
